rpc/ethereum/namespaces/eth/filters/filters.go

```diff
@@ -84,13 +84,12 @@ func newFilter(logger log.Logger, backend Backend, criteria filters.FilterCriter
 }
 
 const (
-	maxFilterBlocks = 100000
-	maxToOverhang   = 600
+	maxToOverhang = 600
```

Comment:
where is this used? should it also be configurable?

Reply:
actually, I'm not sure about the purpose of this logic: https://github.com/tharsis/ethermint/blob/main/rpc/ethereum/namespaces/eth/filters/filters.go#L142, I don't find it in go-ethereum.